app/vmselect/prometheus/prometheus.go

```diff
@@ -1182,17 +1182,16 @@ func getCommonParamsInternal(r *http.Request, startTime time.Time, requireNonEmp
 		return nil, fmt.Errorf("missing `match[]` arg")
 	}
 
-	var filterss [][]storage.TagFilter
+	filterss, err := getTagFilterssFromMatches(matches)
```

Comment:
this rollbacks https://github.com/VictoriaMetrics/VictoriaMetrics/commit/0b7a23a91d5f29535c3c485f0c1d92f87ae91fef change? match[] will be respected in disregard to `ignoreExtraFiltersAtLabelsAPI` setting?

Reply:
Yes, I believe ignoring match[] is incorrect behavior:
 - it breaks compatibility with prometheus API. 
 -  match[] is required param for /api/v1/series.
 - match[] is optional for labels and label_values and not widely used.

 While extra_labels and _extra_filters is VM only settings and you cannot make it optional for grafana variables and completion requests.

Reply:
related issue https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5930